src/main/kotlin/no/nav/sbl/sosialhjelp_mock_alt/datastore/pdl/PdlService.kt

```diff
@@ -298,6 +300,7 @@ class PdlService(
         barnMap[barnFnr] = defaultBarn(etternavn)
 
         pdlAdresseSokService.putAdresse(standardBruker.bostedsadresse.postnummer, standardBruker.bostedsadresse, enhetsnummer)
+        pdlGeografiskTilknytningService.putGeografiskTilknytning(enhetsnummer, standardBruker.bostedsadresse.kommunenummer)
```

Comment:
enhetsnummer?

Reply:
stusset litt på den jeg og nå. sikkert et tegn på at jeg burde gi meg for dagen selv.

Tror enhetsnr og geografisk tilknytning begrepene brukes litt om hverandre i norg-delen av mock-alt. Jeg gir det en ny sjanse i morgen 😄

Reply:
Dersom enhetsnummer er riktig, så burde kanskje argumentet til putGT vært key/id og ikke ident.

Reply:
nei, ser ut som en glipp. Burde være ident :)